examples/certified-counter/src/backend/Cargo.toml

```diff
@@ -15,4 +15,4 @@ ic-cdk-macros = "0.6.0"
 ic-certified-map = "0.3.4"
 serde = "1.0.164"
 serde_cbor = "0.11.2"
-sha256 = "1.1.4"
```

Comment:
For some reason this library depends on `tokio-rs`. After updating packages, `tokio-rs` introduced a breaking change for WASM compilation in a minor version bump :) +1 Rust version control standards are great!!!